Libraries/Image/resolveAssetSource.js

```diff
@@ -104,9 +110,13 @@ function getBasePath(asset) {
  */
 function getScaledAssetPath(asset) {
```

Comment:
@zahanm Because the folder structure as generated by the CLI/bundler also follows the getAssetDestPathAndroid(...) logic. e.g if the image scale is 1, the image will go to the drawable_mdpi folder.